mParticle/aquarium
# fix: style build missing custom rules

## Instructions
1. PR target branch should be against `main`
2. PR title name should follow this format: https://github.com/mParticle/mparticle-workflows/blob/main/.github/workflows/pr-title-check.yml
3. PR branch prefix should follow this format: https://github.com/mParticle/mparticle-workflows/blob/main/.github/workflows/pr-branch-check-name.yml

## Summary
- Fixes custom styles not represented in release 1.8.0

## Testing Plan
- remove dist, src/styles/style.* and src/styles/_variables.css
- run sh scripts/build-dist.sh (this may need to be done twice to generate the new _variables file)
- see that dist has the correct `styles.css` file including all custom rules 

## Reference Issue (For mParticle employees only.  Ignore if you are an outside contributor)
- Closes https://go.mparticle.com/work/UNI-223
